src/components/Footer/Footer.scss

```
@@ -0,0 +1,108 @@
+.footer {
+  display: flex;
+  justify-content: center;
+  background-color: #1c1c1c;
+  color: #999999;
+
+  .footerInner {
+    margin: 0px 160px 0px 160px;
+    padding: 80px 50px 70px 50px;
```

```suggestion
    margin: 0px 160px;
    padding: 80px 50px 70px;
```

margin과 padding은 총 4방향으로 (시계 방향 순서 - 위, 오른쪽, 아래, 왼쪽)
스타일을 지정해 줄 수 있는데, 위 방법과 같이 위와 아래에 지정하는 영역이 같고,
좌우에 지정하는 영역이 같다면 위와 같이 적용해 줄 수 있고,
위, 아래는 다르지만 좌우가 같다면 위와 같이 적용해 줄 수 있습니다.

하지만 위 아래가 같고 좌우가 다른 상황에서는 4방향을 모두 지정해 줘야 합니다